Feat/search

arneroen opened the pull request.

arneroen committed:
feat: implement search

Implement a simple search, that searches only the titles of videos.

arneroen committed:
feat: add tags for searching, expand search.

ArielKarlsen reviewed (changes requested):
The text "Search" on the search button didn't fit in a single line on devices that have more narrow screen. Changing the layout_weight on the searchText to "7.5" and the layout_weight on the searchButton to "2.5" seemed to fix the problem and could be a solution.

arneroen committed:
feat: set width of search field dynamically

set the width of the search button to wrap-content, and set the width
of the search field dynamically based on this.

arneroen commented:
> The text "Search" on the search button didn't fit in a single line on devices that have more narrow screen. Changing the layout_weight on the searchText to "7.5" and the layout_weight on the searchButton to "2.5" seemed to fix the problem and could be a solution.

This does not really fix the problem, it just makes fixes the problem for one specific device. 
Added a commit that sets the width of the button to wrap-content, and then dynamically sets the width of the search field.
